cosmo_tester/test_suites/test_blueprints/hello_world_bash_test.py

```diff
@@ -93,6 +93,10 @@ def _run(self, image_name, user, is_existing_deployment=False):
         floating_ip_id, neutron, nova, sg_id, server_id =\
             self._verify_deployment_installed()
         self.assert_monitoring_data_exists()
+        floating_ip_dict = neutron.show_floatingip(floating_ip_id)
+        expected_output = '{{u\'http_endpoint\': u\'http://{0}:8080\'}}' \
```

Comment:
8080 --> use the property from the server node instead (already done in this very module, you can look how its done there)